superset-frontend/src/explore/components/controls/ConditionalFormattingControl/FormattingPopoverContent.tsx

```diff
@@ -57,6 +57,131 @@ const operatorOptions = [
   { value: COMPARATOR.BETWEEN_OR_RIGHT_EQUAL, label: '< x ≤' },
 ];
 
+const targetValueLeftValidator = (rightValue?: number) => (
+  _: any,
+  value?: number,
+) => {
+  if (!value || !rightValue || Number(rightValue) > Number(value)) {
+    return Promise.resolve();
+  }
+  return Promise.reject(
+    new Error(t('This value should be smaller than the right target value')),
+  );
+};
+
+const targetValueRightValidator = (leftValue?: number) => (
```

Comment:
I might be swatting a fly with a sledge hammer, but it could be nice to have a `targetValueValidator` that encapsulates the repeated logic here.

Reply:
Done